biojava-structure/src/main/java/org/biojava/nbio/structure/io/mmcif/ChemCompGroupFactory.java

```diff
@@ -84,6 +83,13 @@ public static void setChemCompProvider(ChemCompProvider provider) {
 	public static ChemCompProvider getChemCompProvider(){
 		return chemCompProvider;
 	}
+	
+	/**
+	 * Force the cache to be reset.
```

Comment:
This clears the memory cache, right? Could you add that to javadoc, it's an important detail since there is both memory and file cache

Reply:
Only memory. Documented.